Makefile.common

```diff
@@ -0,0 +1,301 @@
+# Copyright 2018 The Prometheus Authors
```

Comment:
2020?

Reply:
That file comes from https://github.com/prometheus/prometheus and will automatically updated whenever the original file changes (based on your amazing script :-p)

Reply:
Seems we will be at 2018 forever :smile: